### DISCUSSION TITLE
peripheral.find("minecraft:chest") not working on server

### DISCUSSION THREAD

**Discussion by Adraah** [cc-tweaked/CC-Tweaked#1216] [Q&A]
As the title says the command peripheral.find("minecraft:chest") is not working in the server and returns a nil value, but I tried the same in a local game and works correctly.
![image](https://user-images.githubusercontent.com/80177793/202065967-5c77450a-4662-4bb7-8f4c-8fe3c8db05f1.png)

**Comment by Lupus590** [comment]
Could you screenshot your code? You might have a typo.

**Reply by SquidDev** [reply]
Might also be worth running the `peripherals` program to get a list of attached peripherals?

**Reply by Adraah** [reply]
This is the code
![image](https://user-images.githubusercontent.com/80177793/202597251-d8c070ec-79d8-4d04-b06f-acfb912557a0.png)

**Reply by Lupus590** [reply]
Your code looks fine, so it's probably that the computer and the chest are not on the same network (or the modem attached to the chest is not configured to treat the chest as a peripheral - right-click with an empty hand on the modem that is connecting to the chest to do this, you'll know that it has worked if you get a chat message with a unique name for that chest).

Double-check that the computer has a modem attached to it, that the chest has one too (chests need full block modems for this to work) and that the two modems are connected together via networking cable (and/or more modems).

If the above doesn't work then run the `peripherals` program and send a screenshot of the output.
